server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -478,8 +479,8 @@ function getLogs() {
 /**
  * REST POST call to clear logs
  */
-function clearLogs() {
-  doLoggedPostCall('/rest/logs/clear', null, false);
+function clearLogs(callback) {
+  doLoggedPostCall('/rest/logs/clear', callback, false);
```

Comment:
Would it be possible to just do this, that way you don't need to pass in the refresh method when calling `clearLogs()`?

```suggestion
function clearLogs() {
  doLoggedPostCall('/rest/logs/clear', refresh, false);
```

Reply:
Addressed in [9ffe0a2](https://github.com/apache/accumulo/pull/2735/commits/9ffe0a2c71cf0e79fe0138001d264a9080ddc3e2)